src/kafka-file/languageservice/services/codeLensProvider.ts

```diff
@@ -114,6 +114,7 @@ export class KafkaFileCodeLenses {
         let keyFormatSettings: Array<SerializationSetting> | undefined;
         let valueFormat;
         let valueFormatSettings: Array<SerializationSetting> | undefined;
+        let headers: Map<String, String> = new Map();

```

Comment:
To be consistent with other values, you not create a new Map.

`let headers: Map<String, String> | undefined;`

Reply:
fixed